templates/ida.tpl

```diff
@@ -0,0 +1,33 @@
+Feature toggles configured for **{{ ida.name }}**:
+--------------------------------------------------
```

Comment:
Careful here, a long IDA name could break the formatting by exceeding the line of dashes.

Reply:
Is there a way to make the header marker dynamic in length?

Reply:
Something like `{% for character in ida.name %}-{% endfor %}`?

Reply:
oh, ok, yeah. I wasn't sure if there was some premade tool for that.

Reply:
That solution ended up being kind of gross, but as long as this is going to always be viewed as rst rendered to html, it should be ok